pkg/model/tangle/solid_entry_points.go

```diff
@@ -66,6 +66,16 @@ func SolidEntryPointsContain(txHash hornet.Hash) bool {
 	return solidEntryPoints.Contains(txHash)
 }
 
+func SolidEntryPointsIndex(txHash hornet.Hash) (bool, milestone.Index) {
```

Comment:
swap bool and index in the return value

Reply:
yes, but those are 2 values we return there, not a "what you requested does not exist" boolean flag